### PR TITLE
[tts][ACR-1716] Add guide for new APIs about service state

### PR DESCRIPTION
**Pull request by stom-hwang**
### Change Description ###

Add guide for new APIs about service state

### API Changes ###

 - ACR: ACR-1716

### CODE REVIEW

File: docs/application/native/guides/text-input/tts.md [hunk 1]
```diff
@@ -327,6 +360,22 @@ To obtain the current state, the supported voice list, and the current voice:
     }
     ```
 
+- Get the current service state using the `tts_get_service_state()` function.
+
```
Comment (syeonholic) [inline docs/application/native/guides/text-input/tts.md:364]: Please add more descriptions of service state in ## TTS information retrieval.

**Review by syeonholic — COMMENTED** [review]
Please check one comment. Thank you!!

File: docs/application/native/guides/text-input/tts.md (hunk 1 above)
Comment (stom-hwang) [reply docs/application/native/guides/text-input/tts.md:364]: Thank you for your opinion.
I fix the guide as similar as csharp one.

**Review by syeonholic — APPROVED** [review]
Thank you for accepting my opinion.
It looks good to me.

**Review by urmain — APPROVED** [review]
It looks good to me.

File: docs/application/native/guides/text-input/tts.md [hunk 2]
```diff
@@ -327,6 +360,22 @@ To obtain the current state, the supported voice list, and the current voice:
...
+    {
+        tts_service_state_e current_state;
+        int ret;
+        ret = tts_get_service_state(tts, &current_state);
+        if (TTS_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
 - Obtain a list of voices supported by the TTS using the `tts_foreach_supported_voices()` function.
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:379]:
```suggestion
- Obtain a list of voices supported by the TTS using the `tts_foreach_supported_voices()` function
```

File: docs/application/native/guides/text-input/tts.md [hunk 3]
```diff
@@ -327,6 +360,22 @@ To obtain the current state, the supported voice list, and the current voice:
     }
     ```
 
+- Get the current service state using the `tts_get_service_state()` function.
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:363]:
```suggestion
- Get the current service state using the `tts_get_service_state()` function
```

**Review by omar-srbd — COMMENTED** [review]
@stom-hwang,
I have checked the files and provided in-line suggestions/corrections wherever I could. However,  there were a few instances where I could not provide suggestions.
Listing them below:
1.	Line 47: “To enable your application to use the TTS functionality:”
Suggestion: “To enable your application to use the TTS functionality, follow the steps below:”
2.	Line 120: “To set and unset callbacks:”
Suggestion: “To set and unset callbacks, follow these steps:”
3.	Line 345: “To obtain the current state, the supported voice list, and the current voice:”
Suggestion: “To obtain the current state, the supported voice list, and the current voice, follow the steps below:”
4.	Line 454: “…such as ringtone or other TTS sounds, are played.”
Suggestion: “…such as ringtones or other TTS sounds, are played.”
5.	Line 486: “To operate TTS:”
Suggestion: “To operate TTS, follow these steps:”
6.	Line 536: “To set and get the options about the TTS engine:”
Suggestion: “To set and get the options about the TTS engine, follow the steps below:”
7.	Line 583: “…maximum limits for the speed, are defined in the…”
Suggestion: “…maximum limits for the speed are defined in the…”
8.	Line 585: “To add text:”
Suggestion: “To add text, follow the steps below:”
9.	Line 632: “To start, pause, and stop the playback:”
Suggestion: “To start, pause, and stop the playback, follow the steps below:”
10.	Removal of period (full stop): 
•	Line 80
•	Line 347
•	Line 402
•	Line 421
•	Line 538
•	Line 553
•	Line 654
•	Line 669

**Comment by stom-hwang** [comment]
Dear @omar-srbd .
Thank you for checking.
I fixed that you mentioned and uploaded it.
And also, I excluded the dotnet guid changes from this PR. It was my mistake.

**Comment by omar-srbd** [comment]
@stom-hwang, thanks for implementing the changes as per suggestion and for removing the _dotnet guide._

@asm-foysal @mijin85cho I am approving this PR for merging.

**Review by omar-srbd — APPROVED** [review]
Approved.

File: docs/application/native/guides/text-input/tts.md [hunk 4]
```diff
@@ -580,9 +630,9 @@ To add text:
 <a name="control"></a>
 ## Control playback
 
-To start, pause, and stop the playback:
+To start, pause, and stop the playback, follow the steps below:
 
-- To start synthesizing the text added in the queue and play the resulting sound data in sequence, use the `tts_play()` function.
+- To start synthesizing the text added in the queue and play the resulting sound data in sequence, use the `tts_play()` function
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:635]:
```suggestion
- To start synthesizing the text added in the queue and play the resulting sound data in sequence, use the `tts_play()` function.
```

File: docs/application/native/guides/text-input/tts.md [hunk 5]
```diff
@@ -602,7 +652,7 @@ To start, pause, and stop the playback:
   }
   ```
 
-- To pause the playback, use the `tts_pause()` function.
+- To pause the playback, use the `tts_pause()` function
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:655]:
```suggestion
- To pause the playback, use the `tts_pause()` function.
```

File: docs/application/native/guides/text-input/tts.md [hunk 6]
```diff
@@ -617,7 +667,7 @@ To start, pause, and stop the playback:
   }
   ```
 
-- To stop the playback, use the `tts_stop()` function.
+- To stop the playback, use the `tts_stop()` function
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/tts.md:670]:
```suggestion
- To stop the playback, use the `tts_stop()` function.
```

**Review by omar-srbd — COMMENTED** [review]
@stom-hwang, please incorporate the suggestions.

**Comment by stom-hwang** [comment]
> @stom-hwang, please incorporate the suggestions.

@omar-srbd Thank you for checking. I updated commit. Please check it again.

**Pull request merged**